[OSDOCS-4880]: [feature] ALB on OSD/ROSA (#62273)
openshift/openshift-docs: networking/aws-load-balancer-operator.adoc

:_content-type: ASSEMBLY
[id="aws-load-balancer"]
= AWS Load Balancer Operator
include::_attributes/common-attributes.adoc[]
:context: aws-load-balancer-operator

toc::[]

The AWS Load Balancer Operator (ALBO) is an Operator supported by Red{nbsp}Hat that users can optionally install on SRE-managed
ifndef::openshift-rosa[]
{product-title}
endif::openshift-rosa[]
ifdef::openshift-rosa[]
{product-title} (ROSA)
endif::openshift-rosa[]

> clusters. The ALBO  manages the lifecycle of the AWS-managed AWS Load Balancer Controller (ALBC) that provisions AWS Elastic Load Balancing v2 (ELBv2) services for applications running in

Reviewer: ```suggestion
clusters. The ALBO manages the lifecycle of the AWS-managed AWS Load Balancer Controller (ALBC) that provisions AWS Elastic Load Balancing v2 (ELBv2) services for applications running in
```

:_content-type: ASSEMBLY
[id="aws-load-balancer"]
= AWS Load Balancer Operator
include::_attributes/common-attributes.adoc[]
:context: aws-load-balancer-operator

toc::[]

> The AWS Load Balancer Operator (ALBO) is a Red-Hat supported Operator that users can optionally install on SRE-managed

Reviewer: ```suggestion
The AWS Load Balancer Operator (ALBO) is a Red Hat supported Operator that users can optionally install on SRE-managed
```
Remove hyphen

ifndef::openshift-rosa[]
{product-title}
endif::openshift-rosa[]
ifdef::openshift-rosa[]
{product-title} (ROSA)
endif::openshift-rosa[]

> clusters.  The ALBO  manages the life cycle of the AWS-managed AWS Load Balancer Controller (ALBC) that provisions AWS Elastic Load Balancing v2 (ELBv2) services for applications running in

Reviewer: ```suggestion
clusters.  The ALBO manages the lifecycle of the AWS-managed AWS Load Balancer Controller (ALBC) that provisions AWS Elastic Load Balancing v2 (ELBv2) services for applications running in
```
Per [the ISG](https://www.ibm.com/docs/en/ibm-style?topic=word-usage#l), `lifecycle` is one word.
